azkaban-web-server/src/main/java/azkaban/webapp/servlet/WebUtils.java

```diff
@@ -93,6 +93,8 @@ public static String formatStatus(final Status status) {
         return "Killing";
       case DISPATCHING:
         return "Dispatching";
+      case POD_FAILED:
```

Comment:
Shall we rename this status to CONTAINER_FAILED? 
^^ cc: @sshardool

Reply:
`CONTAINER_FAILED` certainly seems better than `POD_FAILED` but I was hoping for name that could be use to reflect similar failures for bare-metal executors, e.g when the executor-server crashes. 
Any suggestions @jakhani ?  (or should we not try that at all)